persistent-template/Database/Persist/TH.hs

```diff
@@ -530,19 +511,15 @@ uniqueTypeDec :: MkPersistSettings -> EntityDef -> Dec
 uniqueTypeDec mps t =
     DataInstD [] ''Unique
         [genericDataType mps (entityHaskell t) backendT]
-#if MIN_VERSION_template_haskell(2,11,0)
             Nothing
-#endif
             (map (mkUnique mps t) $ entityUniques t)
             (derivClause $ entityUniques t)
   where
     derivClause [] = []
 #if MIN_VERSION_template_haskell(2,12,0)
     derivClause _  = [DerivClause Nothing [ConT ''Show]]
-#elif MIN_VERSION_template_haskell(2,11,0)
-    derivClause _  = [ConT ''Show]
```

Comment:
Are we only supporting `template-haskell >= 2.12` now?

Reply:
Added a comment on the cabal file - we should add lower bounds if we're dropping CPP support for versions of the libraries

Reply:
`LTS 9.21` has `template-haskell-2.11.1.0`, so I've made the lower bound `>= 2.11`